lib/graphql/lang/ast_reducer.ex

```diff
@@ -0,0 +1,105 @@
+
+# Used for computing a result as a function of the an AST traversal.  The
```

Comment:
It's certainly an interesting question. I haven't looked yet, but I am interested.